VRDR/MortalityData.cs

```diff
@@ -324,1008 +172,15 @@ public string RaceCodeToRaceName(string code)
             {"SD","South Dakota"},
             {"TN","Tennessee"},
             {"TX","Texas"},
+            {"VI","Virgin Islands"},
             {"UT","Utah"},
             {"VT","Vermont"},
-            {"VI","Virgin Islands"},
             {"VA","Virginia"},
             {"WA","Washington"},
             {"WV","West Virginia"},
             {"WI","Wisconsin"},
             {"WY","Wyoming"},
-            {"YC","New York City"},
-            {"TT","Test Jurisdiction"} // This should only be used for testing with NCHS.
```

Comment:
Don't we need TT for testing?

Reply:
It is there....just a github rendering thing.